bwmenu

```diff
@@ -164,22 +188,28 @@ on_rofi_exit() {
 # $1: what to type; all, username, password
 # $2: item name
 auto_type() {
-  item=$(echo "$ITEMS" | jq -r ".[] | select(.name == \"$2\").login")
+  item_array="$(array_from_name "$2")"
   sleep 0.3
```

Comment:
The sleep is only there to give rofi the time to exit before xdotool starts typing in all the keystrokes.
This should be moved to line 198.

Reply:
Got it, makes total sense :100: